### PR TITLE
issue 2015

### PR DESCRIPTION
**Pull request by imjayanthka**
updated http to https

### CODE REVIEW

**Comment by tzarebczan** [comment]
@imjayanthka thanks for the contribution! Can we send your some [LBC in appreciation](https://lbry.io/faq/appreciation)?

**Comment by imjayanthka** [comment]
sure

**Comment by seanyesmunt** [comment]
Awesome! @imjayanthka Could you squash these into one commit? And also add [changelog](https://github.com/lbryio/lbry-desktop/blob/master/CHANGELOG.md) entry?

Then we can merge it 🙂

**Comment by imjayanthka** [comment]
sure give me a sec

**Comment by imjayanthka** [comment]
git commit scripts fail on flow focus-check --color always

**Comment by seanyesmunt** [comment]
I'm guessing it's because some of these files have Flow errors. It's fine to ignore them for this PR, if we want to fix any that would be great too!

For now you can do `git commit -m "update to https" --no-verify`

**Comment by imjayanthka** [comment]
does this work?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 ### Changed
  * Make tooltip smarter ([#1979](https://github.com/lbryio/lbry-desktop/pull/1979))
  * Change channel pages to have 48 items instead of 10 ([#2002](https://github.com/lbryio/lbry-desktop/pull/2002))
-
+ * update to http ([#2016](https://github.com/lbryio/lbry-desktop/pull/2016))
```
Comment (seanyesmunt) [inline CHANGELOG.md]: https*

**Comment by seanyesmunt** [comment]
Yep. Just a small typo

**Comment by tzarebczan** [comment]
@imjayanthka please don't forget to reach out to us so we send you some [LBC in appreciation for the PR](https://lbry.io/faq/appreciation)?

**Pull request merged**